backend/app/services/collections/create_collection.py

```
@@ -71,142 +73,248 @@ def start_job(
 
-    try:
-        with Session(engine) as session:
-            creation_request = CreationRequest(**request)
-            payload = ResponsePayload(**payload)
+    # Keep references for potential backout/cleanup on failure
+    assistant = None
+    assistant_crud = None
+    vector_store = None
+    vector_store_crud = None
+    collection_job = None
+    callback = None
```

Ruff (0.14.3)

Local variable `callback` is assigned to but never used

Remove assignment to unused variable `callback`

(F841)